packages/web-react/src/components/Pagination/PaginationLinkDirectional.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ElementType, ForwardedRef, forwardRef } from 'react';
+import { SpiritPaginationLinkDirectionalProps } from '../../types';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+
+/* We need an exception for components exported with forwardRef */
+/* eslint no-underscore-dangle: ['error', { allow: ['_PaginationLinkDirectional'] }] */
+const _PaginationLinkDirectional = <E extends ElementType = 'a'>(
```

Comment:
I came across multiple Pagination components using components.gallery and still thinking about the `LinkDirectional` name. In most cases, there was `ButtonLink`, `PaginationButton`, `PaginationLink`, or something similar.

This makes me think about my previous [comment](https://github.com/lmc-eu/spirit-design-system/pull/879/files#r1223892777) to this file, if we use `ButtonLink` instead of `Button` that maybe the name `PaginationButton`, eg. `PaginationButtonLink` will suit more 🤔 

I am not quite sure about this so I leave the decision up to you.